Content.Shared/Terminator/Components/TerminatorComponent.cs

```diff
@@ -0,0 +1,56 @@
+using Content.Shared.Actions;
+using Content.Shared.Roles;
+using Content.Shared.Terminator.Systems;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Terminator.Components;
+
+/// <summary>
+/// Main exterminator component, handles the target, if any, and adding the curse action.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, Access(typeof(SharedTerminatorSystem))]
+public sealed partial class TerminatorComponent : Component
+{
```

Comment:
What heppened to calling it exterminator to avoid any licensing bs.

Reply:
it was Terminator internally already but i can fully rename everything :trollface:

Reply:
ok mickey mouse has been fended off everything has been renamed